packages/api/models/Operator.ts

```diff
@@ -20,19 +13,18 @@ export enum AppEnvAccess {
 export interface Operator {
   display_name: string;
   email: string;
-  access: AccessLevel;
+  access: string;
```

Comment:
I believe this `access` can be removed since you are extending `Operator` and adding `access` in `PortfolioOperator` and `AppEnvOperator`.

Reply:
Looks like this was addressed.